script/cms/compare-json.js

```diff
@@ -0,0 +1,322 @@
+  },
+  {
+    name: 'bundle',
+    type: String,
+    description:
+      'Compare all entities of the bundle type specified. (Only node entities are supported) i.e. node-q_a, node-health_care_local_facility, etc',
+    multiple: true,
+  },
+  {
+    name: 'help',
+    type: Boolean,
+    description: 'Show this help menu.',
+  },
+];
+
+const { entity: entityNames, bundle, help } = commandLineArgs(commandLineDefs);
+
+if (help) {
+  console.log(
+    commandLineUsage([
+      {
+        header: 'Content Build JSON Comparison',
+        content:
+          'Compare the output of GraphQL and transformed CMS export data',
+      },
+      {
+        header: 'Options',
+        optionList: commandLineDefs,
+      },
+    ]),
+  );
+  process.exit(0);
+}
+
+/**
+ * Converts deep-diff's 'kind' property
+ * @param {string} kind - The string representing the kind of difference
+ * @return {string} Description of the kind of difference
+ */
+const getDiffType = kind => {
+  switch (kind) {
+    case 'A':
+      return 'Array Change';
+    case 'D':
+      return 'Property Deleted';
+    case 'E':
+      return 'Property Edited';
+    case 'N':
+      return 'New property Added';
+    default:
+      return 'Change';
+  }
+};
+
+/**
+ * Converts deep-diff's 'item' property
+ * https://www.npmjs.com/package/deep-diff#differences
+ * @param {object} item - A deep-diff 'item' object
+ * @return {object} A modified deep-diff 'item' object
+ */
+const getDiffItem = item => {
+  const diffItem = {
+    item: {
+      diffType: getDiffType(item.kind),
+    },
+  };
+  if (!item.lhs || !item.rhs) {
+    return !item.lhs
+      ? { ...diffItem.item, cmsExport: item.rhs }
+      : { ...diffItem.item, graphQL: item.lhs };
+  }
+  return {};
+};
+
+/**
+ * Finds the parent node and path in ancestor node(s)
+ * of non node entities. Only some paragraph entities are
+ * currently supported
+ * @param {object} node - The node to find the parent of
+ * @param {object []} rawEntities - Array of raw entities
+ * @param {object} parent - The object containing the path and parent node
+ * @return {array} The finalized array containing the path and parent object of the node
+ */
+const getParentNode = (
+  node,
+  rawEntities,
+  parent = { path: '', node: null },
+) => {
+  if (node.baseType === 'node') {
+    // eslint-disable-next-line no-param-reassign
+    parent.node = node;
+  }
+
+  if (parent.path === '') {
+    if (node.parent_type[0].value === 'node') {
+      // eslint-disable-next-line no-param-reassign
+      parent.path += node.parent_field_name[0].value;
+      const ent = rawEntities.find(
+        entity =>
+          entity.baseType === 'node' &&
+          parseInt(entity.nid[0].value, 10) ===
+            parseInt(node.parent_id[0].value, 10),
+      );
+      getParentNode(ent, rawEntities, parent);
+    }
+  } else {
+    // TODO: Add support for deeper nested paragraphs, and other entity types
```

Comment:
@theodur  Do we need this for initial release? If not, should we make a ticket for it?

Reply:
I don't think we need this for the initial release. I can make a ticket for it and add it to the backlog.